libs/base/ui/form/multi-range/src/multi-range.model.ts

```diff
@@ -41,3 +41,8 @@ export interface MultiRangeProperties {
 
   disabled?: boolean;
 }
+
+export interface MultiRangeChangeEvent {
+  minValue: number;
+  maxValue: number;
```

Comment:
It will be a BC. I've just provided missed public interface for the `CustomEvent`, these field names are already part of Alexander's implementation of multi-range component

Reply:
ok, we'll keep it as-is